printing_templates/One Dive.html

```diff
@@ -4,104 +4,141 @@
 		body {
 			{{ print_options.grayscale }};
 			padding: 0;
-			margin: 0;
+			margin: 0 0 0 6%;  <!-- Provide LH margin for binding the page -->
 			font-size: {{ template_options.font_size }}vw;
 			line-height: {{ template_options.line_spacing }};
 			font-family: {{ template_options.font }};
 		}
 
 		h1 {
-			float: left;
 			font-size: {{ template_options.font_size }}vw;
 		}
 
 		p {
-			float: left;
 			font-size: {{ template_options.font_size }}vw;
 		}
 
 		table {
 			-webkit-box-sizing: border-box;
 			box-sizing: border-box;
-			border-width: {{ template_options.borderwidth }}px;
+			//border-width: {{ template_options.borderwidth }}px;
+			border-width: 2px;
 			border-style:solid;
 			border-color: {{ template_options.color6 }};
+			border-spacing: 2px;
+			border-collapse: separate;
+		}
+
+		tr {
+			height: 40px;
```

Comment:
sadly, `px` are non-portable from my tests. they kind of work for borders and lines, but when you start using them for `height` and padding and you start getting different results on different page sizes. can you please, test this on A5, A4, A3, A2..etc and see if this value is still good and see if you are happy with it?

to the question "what else can we use if not using px?" the answer is that the only good unit is `%` as we want the templates to work on any page size.

Reply:
i've linked this in a comment bellow:
https://stackoverflow.com/questions/16056591/font-scaling-based-on-width-of-container

it applies to some extent here, as well. the thing is, from my tests some of these alternative units simply do not work with the WebKit / Grantlee solution, which we are using.